packages/linter/src/executors/eslint/lint.impl.ts

```diff
@@ -38,6 +38,20 @@ export default async function run(
 
   const eslint = new projectESLint.ESLint({});
 
+  /**
+   * Entire folder might be excluded via .eslintingore which would result
+   * in `lintResults` being an empty array.
+   * This would result in false assumption of invalid configuration and
+   * end up as exception.
+   *
+   * We want to rather show info as this is likely intentional behavior
+   */
+  const projectPath = context.workspace.projects[context.projectName].root;
+  if (await eslint.isPathIgnored(`${context.root}/${projectPath}/**/*`)) {
```

Comment:
I think if we remove this https://github.com/nrwl/nx/blob/master/packages/linter/src/executors/eslint/utility/eslint-utils.ts#L38.. and then catch the error in the implementation... it will handle the case we want.. without having to check ignored patterns

Reply:
Not really. The empty results (or error if we set `errorOnUnmatchedPattern` to true) can come in two scenarios:
- Project is ignored via `.eslintignore` 
- Project has invalid configuration in `workspace/angular.json`

If we set `errorOnUnmatchedPattern` to true, the result would be the same, just exception message would be different. Without any catching the error is e.g.:
```
No files matching 'apps/api/**/*.ts' were found.
```

We want to throw exception only in the second case, so we need to check whether empty result (or exception) was caused by invalid config or exclusion of the folder. And the only way to be sure is to check whether the project folder is not ignored.